frontends/web/package.json

```diff
@@ -3,12 +3,12 @@
   "name": "BitBoxApp",
   "version": "0.0.0",
   "scripts": {
-    "postinstall": "npx tcm src --camelCase dashes",
-    "build": "preact build --no-prerender --service-worker=false",
-    "dev": "preact watch",
-    "lint": "eslint --max-warnings 0 --cache --ignore-path ../../.gitignore '**/*.{js,jsx}'",
-    "postlint": "tslint -e 'src/**/*.css.d.ts' 'src/**/*.ts?(x)'",
-    "test": "jest",
+    "gen-ts-css": "npx tcm src --camelCase dashes",
```

Comment:
no need for using `npx` here, as `tcm` will  be installed with `typed-css-modules` and is "like" globally available from within package.json as it is in `node_modules/.bin/tcm -> ../typed-css-modules/lib/cli.js`

https://github.com/Quramy/typed-css-modules/blob/3b9f9e55993b28f9946b2573053bf30a240d9731/package.json#L16

just `"tcm src --camelCase dashes"` is enough

Reply:
nice, fixed (also for `npx browserlist`).

Reply:
nice